copier.yml

```
@@ -179,6 +179,50 @@ install_as_windows_service:
+    when: "{{ install_as_windows_service }}"
+
+installer_upgrade_code:
+    type: str
+    help: "A fixed-for-life UpgradeCode GUID for this app's MSI (generate once, NEVER change it - it identifies upgrades across versions). Format: 8-4-4-4-12 hex, e.g. 3b9d1f6a-2c84-4e7b-9a1f-6d5c4b3a2e10."
+    when: "{{ install_as_windows_service }}"
+    validator: "{% if not (installer_upgrade_code | regex_search('^[0-9a-fA-F]{8}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{12}$')) %}Must be a GUID like 3b9d1f6a-2c84-4e7b-9a1f-6d5c4b3a2e10{% endif %}"
+
+sign_installer:
+    type: bool
+    help: Should the MSI be code-signed via Azure Trusted Signing during CI?
+    default: no
```

YAMLlint (1.37.1)

[warning] truthy value should be one of [false, true]

(truthy)